src/Models/Speaker.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Models;
+
+use Blumilk\Meetup\Core\Formats;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
```

Comment:
```suggestion
```
`CamelCaseAttributes` trait was not working. Speaker model was extending default Eloquent model.